pyvista/plotting/plotter.py

```diff
@@ -7192,7 +7192,11 @@ def meshes(self) -> list[pyvista.DataSet | pyvista.MultiBlock]:  # numpydoc igno
             List of mesh objects such as pyvista.PolyData, pyvista.UnstructuredGrid, etc.
 
         """
-        return [actor.mapper.dataset for actor in self.actors.values() if hasattr(actor, 'mapper')]
+        return [
+            actor.mapper.dataset
+            for actor in self.actors.values()
+            if hasattr(actor, 'mapper') and hasattr(actor.mapper, 'dataset')
+        ]
```

Comment:
I think this is just addressing the symptom rather than the underlying issue. I'll take a look at this specific issue later and validate what's going wrong here